default_app/main.js

```diff
@@ -253,13 +253,16 @@ function loadApplicationPackage (packagePath) {
   } catch (e) {
     if (e.code === 'MODULE_NOT_FOUND') {
       app.focus()
-      dialog.showErrorBox(
-        'Error opening app',
-        'The app provided is not a valid Electron app, please read the docs on how to write one:\n' +
-        `https://github.com/electron/electron/tree/v${process.versions.electron}/docs
-
-${e.toString()}`
-      )
+      dialog.showMessageBox({
+        message: 'Error opening app',
+        detail: 'Unable to open or find an Electron app. Click to learn more on how to write an app.\n\n' +
+            `${e.toString()}`,
+        buttons: ['Learn More', 'OK']
+      }, (response) => {
+        if (response === 0) {
+          shell.openExternal(`https://github.com/electron/electron/tree/v${process.versions.electron}/docs`)
```

Comment:
I think it might make sense to change this URL to http://electron.atom.io/docs now